.test_durations

```diff
@@ -968,16 +968,3389 @@
     "tests/aws/services/lambda_/test_lambda_developer_tools.py::TestHotReloading::test_hot_reloading[nodejs18.x]": 3.06303496099963,
     "tests/aws/services/lambda_/test_lambda_developer_tools.py::TestHotReloading::test_hot_reloading[python3.9]": 2.900148914000056,
     "tests/aws/services/lambda_/test_lambda_developer_tools.py::TestHotReloading::test_hot_reloading_publish_version": 0.17520181300005788,
-    "tests/aws/services/lambda_/test_lambda_integration_dynamodbstreams.py::TestDynamoDBEventSourceMapping::test_deletion_event_source_mapping_with_dynamodb": 6.989548434999961,
```

Comment:
This one seems suspicious 🤔 Did it get more testing added to it?

Reply:
Could you add a bit more context please? Why is it suspicious? What do you mean by "did it get more testing added to it"?

Reply:
It went from roughly 7 seconds to 65 seconds. That's quite the increase. I was wondering if we have some kind of performance degradation here, or if the test was extended so that it takes 10 times longer now.

Reply:
Well, that's hard to say honestly. We'll see the results in the upcoming pipeline runs. If the data is too far off, we would see that in the balancing (that they are still not well balanced). Then we can dig deeper.